player/lua/stats.lua

```diff
@@ -719,8 +719,8 @@ local function add_video(s)
         append(s, scaled_height, {prefix="x", nl="", indent=" ", prefix_sep=" ", no_prefix_markup=true})
     end
     append_property(s, "current-window-scale", {prefix="Window Scale:"})
-    if r["aspect"] ~= nil then
-        append(s, format("%.2f", r["aspect"]), {prefix="Aspect Ratio:"})
+    if r["aspect-name"] ~= nil then
+        append(s, r["aspect-name"], {prefix="Aspect Ratio:"})
```

Comment:
IMO, aspect num should be showed always.
`VistaVision Ratio` is quite obscure for me. I know 3:2 because it is the popular aspects for some pads. Never hear about VistaVision before.
`aspect-name` can be append.

Reply:
It will be shown always, name would be `1.5:1 (VistaVision Ratio)`. No information is lost. Also it is nice information to show those unknown for most names of the format that they might be watching.

Reply:
I probably should split the string tbh.

Reply:
> Also it is nice information to show those unknown for most names of the format that they might be watching

I'm not sure if it was the truth.
Today I prefer to believe a 3:2 video is made on Surface instead of the ancient VistaVision format.

Reply:
and split, for more flexibility on user side.

Reply:
>  Today I prefer to believe a 3:2 video is made on Surface instead of the ancient VistaVision format.

Well I use mpv to watch movies, and movies are timeless, at least some of them. I don't watch MS Surface adverts.

Panavision is also ancient, applying your logic, yet most movies use the 2.39:1 format.

Reply:
> I don't watch MS Surface adverts.

Not adverts. Surface here is just a example. You could find more and more such devices use this aspect.
Today some aspects ration were widely used because of new devices' popularity.
Any video/photo created by ipad(11incn)'s aspect can be 3:2. Obviously Apple has nothing to do with visionvista.

Assuming that When I watch the videos recorded on my ipad by mpv. I see the info say it is visionvista, I just feel confused and strange.

Reply:
There is no real "names". Aspect ratio is just a ratio of width to height, depending on the context you may call it differently. I like movie theme based naming, as a format in which movies are produced. I will not call 3:2 "Apple ratio", because it is something they use, not introduced. Unlike mentioned movie formats that were at the time pioneers in the area. You know in analog era it was a big deal to introduce new format, now everything is 16:9 in consumer space anyway with black bars...